docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -153,6 +181,12 @@ Advanced Vagrant
 The following steps are all optional, so feel free to pick and choose which you would like to
 follow.
 
+#. If your provisioning of the vagrant box fails or you make modifications to the scripts
+   or ansible roles that provision the vagrant box, you must re-run the provisioning in vagrant
+   because it will only run once.
+
+      $ vagrant provision
```

Comment:
This command should be escaped. Similar `::` above. I wish I know how to not use the `::` and still have it escape.